package.json

```diff
@@ -20,12 +20,13 @@
   "devDependencies": {
     "ava": "^0.18.0",
     "debug": "^3.1.0",
-    "nock": "^9.1.4",
+    "nock": "^11.7.0",
```

Comment:
The test in this PR was *also* failing because `nock` itself was using private request header APIs in version 9.